### PR TITLE
Feature/finos#351 block div

### PR DESCRIPTION
**Pull request by wintersoil**
# Terms

> THIS SOFTWARE IS CONTRIBUTED SUBJECT TO THE TERMS OF THE TERMS OF THE CCLA DATED 2017-11-07 WITH FINOS/LINUX FOUNDATION (FORMERLY THE SYMPHONY SOFTWARE FOUNDATION CCLA).
>
> THIS SOFTWARE IS LICENSED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE AND ANY WARRANTY OF NON-INFRINGEMENT, ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. THIS SOFTWARE MAY BE REDISTRIBUTED TO OTHERS ONLY BY EFFECTIVELY USING THIS OR ANOTHER EQUIVALENT DISCLAIMER IN ADDITION TO ANY OTHER REQUIRED LICENSE TERMS.

### CODE REVIEW

File: src/Morphir/Visual/ViewValue.elm [hunk 1]
```diff
@@ -120,7 +120,7 @@ viewValueByLanguageFeature config value =
                     ViewList.view config (viewValue config) itemType items
 
                 Value.Variable tpe name ->
-                    el []
+                    el [ width Element.fill, center]
```
Comment (AttilaMihaly) [inline src/Morphir/Visual/ViewValue.elm:123]: Let's be consistent and use `fill` instead of `Element.fill`.

**Review by AttilaMihaly — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -62,8 +62,10 @@
   },
   "dependencies": {
     "commander": "^4.1.1",
+    "elm-live": "^4.0.2",
     "express": "^4.17.1",
     "fs-extra": "^9.0.1",
-    "get-stdin": "^8.0.0"
+    "get-stdin": "^8.0.0",
+    "morphir-insight-api": "^1.0.0"
```
[the remark this answers is not in the document]
Comment (wintersoil) [reply package.json]: Comment addressed

File: src/Morphir/Visual/Components/DecisionTree.elm [hunk 3]
```diff
@@ -60,6 +58,19 @@ highlighStateToColor state =
...
 
+highlightStateToBackground : HighlightState -> String
+highlightStateToBackground state =
+    case state of
+        Highlighted bool ->
+            if bool then
+                "none"
+
+            else
+                "repeating-linear-gradient( 45deg, #606dbc, #606dbc 10px, #465298 10px, #465298 20px)"
```
[the remark this answers is not in the document]
Comment (wintersoil) [reply src/Morphir/Visual/Components/DecisionTree.elm]: Comment addressed => changed to grayscale

File: src/Morphir/Visual/ViewValue.elm (hunk 1 above)
Comment (wintersoil) [reply src/Morphir/Visual/ViewValue.elm]: Comment addressed

File: src/Morphir/Visual/ViewApply.elm [hunk 4]
```diff
@@ -47,32 +48,50 @@ view config viewValue functionValue argValues =
...
+                                [ smallSpacing config.state.theme |> spacing ]
+                                [ viewValue functionValue
+                                , column
+                                    [ mediumPadding config.state.theme |> padding
+                                    , smallSpacing config.state.theme |> spacing
+                                    ]
+                                    (argValues
+                                        |> List.map viewValue
+                                    )
+                                ]
```
[the remark this answers is not in the document]
Comment (wintersoil) [reply src/Morphir/Visual/ViewApply.elm:93]: Comment not addressed => not sure how to redirect to existing logic

**Review by AttilaMihaly — APPROVED** [review]

**Review by AttilaMihaly — APPROVED** [review]

**Pull request merged**